docs/source/concepts/functionality/interfaces/yaml_based/dynamic_sectors.rst

```diff
@@ -0,0 +1,52 @@
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
+
+.. _dynamic_sectors_functionality:
+
+*************************
+Dynamic Sectors in GeoIPS
+*************************
+
+A dynamic sector is a YAML-based GeoIPS plugin that contains the information
+needed to generate a geographic sector for which data will be plotted. Dynamic
+sectors include information about the intended size, resolution, and projection
+of the resulting geographic sector, but leave the specific location to be
+calculated at run time based on the data being used.
+
+Dynamic sectors are most commonly used when plotting tropical cyclone imagery,
+as they allow for a sector that can "follow along" with the storm.

```

Comment:
Maybe make a note that these could be used for other intermittent environmental or atmospheric events?  Active volcanoes, atmospheric rivers, PyroCumulonimbus, etc.  And dynamic sectors could include different types of information, and different fields could be dynamic for different dynamic sectors (ie, you could specify the full min/max lat/lon and resolution rather than only specifying the center as is done for TCs).

Just to make it clear that things could be set up for dynamic processing of other things besides just TCs